modules/300-prometheus/templates/prometheus/prometheus.yaml

```diff
@@ -150,11 +150,7 @@ spec:
 {{- end }}
   scrapeInterval: {{ .Values.prometheus.scrapeInterval | default "30s"}}
   evaluationInterval: {{ .Values.prometheus.scrapeInterval | default "30s" }}
-  externalLabels:
-{{- with .Values.prometheus.externalLabels }}
-    {{- . | toYaml | nindent 4 }}
-{{- end }}
-    prometheus: deckhouse
+  externalLabels: {}
```

Comment:
We can omit this change, right?

Reply:
Unfortunately, no. To prevent adding this field in the future, Alexey added the comment.